libvast/vast/system/actors.hpp

```diff
@@ -281,6 +281,10 @@ using exporter_actor = typed_actor_fwd<
   // Register the SINK actor.
   caf::reacts_to<atom::sink, caf::actor>,
   // Register a list of IMPORTER actors.
+  // FIXME: This cannot take an importer_actor, because that'd introduce a
+  // cyclic dependency between the importer_actor and exporter_actor
+  // declarations. We should improve the logic of wiring up the importers with
+  // continuous exporters, e.g., by having the node deal with the issue.
```

Comment:
I implemented a solution to this. Am I allowed to push on this branch?

Reply:
Yes, I just turned off auto-merge.

Reply:
I just did, let me know what you think.

Reply:
Yes, that works.

I'll do another pass to fixup the naming so it's more consistent.

Reply:
Also just noticed that your change broke CI, so you must've missed some non-typed actor messaging.